CMakeLists.txt

```diff
@@ -53,8 +53,7 @@ option(WITH_COVERAGE    "Compile PaddlePaddle with code coverage"       OFF)
 option(COVERALLS_UPLOAD "Package code coverage data to coveralls"       OFF)
 option(ON_TRAVIS        "Exclude special unit test on Travis CI"        OFF)
 option(WITH_C_API       "Compile PaddlePaddle with C-API(Prediction)"   OFF)
-# TODO: Only compile PaddlePaddle fluid version by WITH_FLUID option. 
-option(WITH_FLUID       "Compile PaddlePaddle fluid only(TODO)"         OFF)
+option(WITH_ONLY_FLUID  "Compile PaddlePaddle fluid only"               OFF)
```

Comment:
`WITH_FLUID_ONLY` might make more sense?

Reply:
@abhinavarora You are right, I change to `WITH_FLUID_ONLY` now.